pkg/controller/migmigration/task.go

```diff
@@ -1362,6 +1379,66 @@ func (t *Task) hasImageStreams() (bool, error) {
 	return false, nil
 }
 
+// Get whether the associated plan has relevant artifacts using the analytics resource.
+func (t *Task) hasRelevantArtifacts() (bool, string, error) {
+	noMsg := ""
+	client, err := t.getDestinationClient()
+	if err != nil {
+		log.Trace(err)
+		return false, noMsg, err
+	}
+
+	analytic := migapi.MigAnalytic{}
+
+	err = client.Get(context.Background(),
+		k8sclient.ObjectKey{
+			Name:      t.PlanResources.MigPlan.Name,
+			Namespace: t.PlanResources.MigPlan.Namespace,
+		},
+		&analytic)
+	if err != nil {
+		log.Trace(err)
```

Comment:
Instead of doing `log.Trace(err)` anywhere in this PR, we should be wrapping and returning errors.

```
# Convert this...
log.Trace(err)
return false, noMsg, err

# To this
return false, noMsg, liberr.wrap(err)
```

log.Trace() is only used if we are in the top level reconcile() function for one of the controllers.